### PR TITLE
xddlc-2021-finalproject-readme file

### PR DESCRIPTION
**Pull request by xddlc**
My own conclusion of what I had learnt

### CODE REVIEW

**Comment by xddlc** [comment]
I had commit.

**Comment by xddlc** [comment]
@MuZhou233

File: 2021-finalproject/Bash-file [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```
Comment (MuZhou233) [inline 2021-finalproject/Bash-file:1]: Is this script write by yourself?

File: 2021-finalproject/README2.md [hunk 2]
```diff
@@ -0,0 +1,385 @@
+# shell的简单命令
```
Comment (MuZhou233) [inline 2021-finalproject/README2.md:1]: This file name is meaningless, try a meaningful name

**Review by MuZhou233 — CHANGES_REQUESTED** [review]

File: 2021-finalproject/README2.md (hunk 2 above)
Comment (xddlc) [reply 2021-finalproject/README2.md]: I had changed the meaningless name. Thank you!

File: 2021-finalproject/Bash-file (hunk 1 above)
Comment (xddlc) [reply 2021-finalproject/Bash-file]: OK.That was not written by myself. I had delete it.And surely I don't know how to write the scrip.I'm sorry! And I will try my best to learn it!

**Comment by xddlc** [comment]
@MuZhou233

**Review by xddlc — COMMENTED** [review]
I had committed the newest finalproject

**Review by xddlc — COMMENTED** [review]
I had changed.

**Review by xddlc — COMMENTED** [review]
....

**Comment by xddlc** [comment]
..

**Comment by MuZhou233** [comment]
You should pass the check action first

**Comment by xddlc** [comment]
Is it OK now?

**Review by xddlc — COMMENTED** [review]
delete

**Pull request closed without merging**